Source/WebCore/PAL/pal/spi/ios/MobileGestaltSPI.h

```diff
@@ -95,6 +95,8 @@ SInt32 MGGetSInt32Answer(CFStringRef question, SInt32 defaultValue);
 Float32 MGGetFloat32Answer(CFStringRef question, Float32 defaultValue);
 #endif
 
+int MGSetAnswer(CFStringRef question, CFTypeRef answer);
```

Comment:
Do we need to guard for any particular OS versions? I suspect this is present on all supported iOS targets, but wanted to double-check.